pkg/controller/iperf/iperf_controller.go

```diff
@@ -124,6 +124,10 @@ func (r *ReconcileIperf) Reconcile(request reconcile.Request) (reconcile.Result,
 		return reconcile.Result{}, err
 	}
 
+	if len(workerNodeList.Items) == 0 {
+		return reconcile.Result{}, fmt.Errorf("could not find at least one worker node")
```

Comment:
Would we expect the issue to be resolved? If not we return `reconile.Result{Requeue: false}` here so that we don't hot loop and error for ever.

Reply:
Updated the PR. Thanks!